src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -1490,6 +1490,11 @@ def handle_report_tactical_command(
         conditions = tactical_report.details.get("conditions")
         actions = tactical_report.details.get("actions")
         needs = tactical_report.details.get("needs")
+    else:
```

Comment:
I think we should add open status tasks to the actions field regardless of whether we have a tactical report or not correct?

Reply:
Also, we should make use of the `TaskStatus` enum `TaskStatus.resolved` instead of hardcoding `Resolved`